src/components/Carousel.js

```diff
@@ -115,6 +115,29 @@ class Carousel extends Component {
         this.destroyCarousel();
     }
 
+    setThumbsRef = e => {
```

Comment:
`e` sounds like `event`. We could change this to `node` or `thumbsNode`. What do you think? The same applies for the other refs

Reply:
Ah yes, I am used to named the variable e (which stands for element), but I think `node` will indeed be better to avoid confusion. Will change this.